### ISSUE TITLE
No space between cookies specified on the command line

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#152]
```
My team ran into some issues with a server thinking that sessions had expired. 
It turned out that skipfish's client isn't including a space in the separator 
between cookies.

So skipfish was giving a header like

Cookie: COOKIE1=123;COOKIE2=456

But this was being interpreted as a single cookie "COOKIE1=123;COOKIE2"

I looked up the RFC, and according to section 4.2.1 of RFC 6265, the user agent 
should separate each cookie pair with a semicolon followed by a space.

Adding a space in front of the cookie name, like 
-C " COOKIE1=123" -C " COOKIE2=456"
seems to work.
```

Original issue reported on code.google.com by `chanalde...@gmail.com` on 3 May 2012 at 6:34

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report! Next version (2.06b) will have a space.  

```

Original comment by `niels.he...@gmail.com` on 4 May 2012 at 11:37
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.06b. Thanks for the report!
```

Original comment by `niels.he...@gmail.com` on 13 May 2012 at 10:41
- Changed state: **Fixed**
